### PR TITLE
Bump pg_auto_failover version to 1.4.2

### PR DESCRIPTION
**Pull request by gurkanindibay**

### CODE REVIEW

File: debian/changelog [hunk 1]
```diff
@@ -1,3 +1,9 @@
+pg-auto-failover (1.4.2-1) stable; urgency=low
```
Comment (hanefi) [inline debian/changelog]:
```suggestion
pg-auto-failover (1.4.2-2) stable; urgency=low
```

Let's bump the fancy version. Maybe we can add a changelog item as well

File: debian/changelog [hunk 2]
```diff
@@ -1,3 +1,9 @@
...
+
+  * Official 1.4.2 release of pg_auto_failover
```
Comment (hanefi) [inline debian/changelog]:
```suggestion
  * Official 1.4.2 release of pg_auto_failover
  * Adds the missing dependency postgresqlXX-contrib
```

File: pg-auto-failover.spec [hunk 3]
```diff
@@ -82,6 +82,9 @@ PATH=%{pginstdir}/bin:$PATH
 
 
 %changelog
+* Thu Feb 11 2021 - Gurkan Indibay <gindibay@microsoft.com> 1.4.2-1
```
Comment (hanefi) [inline pg-auto-failover.spec]:
```suggestion
* Thu Feb 11 2021 - Gurkan Indibay <gindibay@microsoft.com> 1.4.2-2
```

File: pg-auto-failover.spec [hunk 4]
```diff
@@ -82,6 +82,9 @@ PATH=%{pginstdir}/bin:$PATH
...
+- Official 1.4.2 release of pg_auto_failover
```
Comment (hanefi) [inline pg-auto-failover.spec:86]: If you decide to add a changelog item, please place the same entry here as well.

**Review by hanefi — CHANGES_REQUESTED** [review]
Please remove Debian based distros from the CI build matrix in a separate commit. On future releases we can revert that commit to get back to the original list of distros.

File: debian/changelog (hunk 2 above)
Comment (gurkanindibay) [reply debian/changelog]: We will not release for  debiam so I think we don't need to change the changelog

File: debian/changelog (hunk 1 above)
Comment (gurkanindibay) [reply debian/changelog]: I removed the unnecessary entry because there is no need to release for debian

File: pg-auto-failover.spec (hunk 3 above)
Comment (gurkanindibay) [reply pg-auto-failover.spec]: Fixed it

**Review by hanefi — APPROVED** [review]

**Pull request merged**